test/wdio.conf.ts

```diff
@@ -91,12 +92,13 @@ export const config: Options.Testrunner = {
     //
     // Place inside the array to run sequentially.
     [
-      './test/specs/**/*.e2e.ts'
+      // './test/specs/**/*.e2e.ts'
 
       // './test/specs/**/anInitialSuite.e2e.ts',
       // './test/specs/**/apexReplayDebugger.e2e.ts',
       // './test/specs/**/authentication.e2e.ts',
       // './test/specs/**/debugApexTests.e2e.ts',
+      './test/specs/**/lwcLsp.e2e.ts',
```

Comment:
Can you revert the changes from this file? I think these can probably stay the same, as @jeffb-sfdc's changes will mod these later.

Reply:
Sure, this was just while I was working on this